client/__tests__/util/centroid.test.ts

```diff
@@ -31,7 +31,11 @@ describe("centroid", () => {
       schema,
       "field4",
       obsAnnotations,
```

Comment:
Optional:  given the confusion around the type of an obsAnnnotation, you may want to add tests which exercise the other label types (eg, number or boolean or blended)